src/js/components/homepage/download/DownloadDetail.jsx

```diff
@@ -62,6 +63,17 @@ export default class DownloadDetail extends React.Component {
                 </button>
             );
         }
+        else if (this.props.internalDomain) {
```

Comment:
we should be using the `<Link />` element from React Router for internal navigation?